.github/workflows/bundle-size/status-report.js

```diff
@@ -26,7 +26,7 @@ function getStatusMetadata(context) {
   return {
     owner: context.repo.owner,
     repo: context.repo.repo,
-    sha: context.payload.pull_request.head.sha,
```

Comment:
Use `sha` directly from the context: https://github.com/actions/toolkit/blob/main/packages/github/src/context.ts#L13

This allows it to work both for pushes and for pull requests.